experiment/textcov.py

```diff
@@ -257,12 +271,24 @@ def from_jvm_file(cls, file_handle) -> Textcov:
         for method_item in item:
           if method_item.tag == 'method':
             if method_item.attrib['name'] not in JVM_SKIPPED_METHOD:
-              class_method_items.append((class_name, method_item))
+              class_method_items.append((class_name, method_item, coverage))
 
-    for class_name, method_item in class_method_items:
+    for class_name, method_item, coverage in class_method_items:
       method_dict = method_item.attrib
       method_name = method_dict['name']
 
+      # Deteremine start index in coverage list
+      start_line = int(method_dict['line'])
```

Comment:
Instead of type casting here can we do it when we store the line number?

Reply:
The number is not stored before, the method_dict is lively retrieved on line 277 from the XML ET.

Reply:
Ah sorry, for some reason I thought we assembled this dict ourselves.